lib/foreman_openscap/engine.rb

```diff
@@ -32,41 +32,59 @@ class Engine < ::Rails::Engine
                                        'foreman_openscap/policy.css']}}
     end
 
-    initializer 'foreman_openscap.register_plugin', :after=> :finisher_hook do |app|
+    initializer 'foreman_openscap.register_plugin', :after => :finisher_hook do |app|
       Foreman::Plugin.register :foreman_openscap do
         requires_foreman '>= 1.5'
 
         # Add permissions
         security_block :foreman_openscap do
-          permission :view_arf_reports, {:scaptimony_arf_reports => [:index, :show],
-                                         :scaptimony_policies => [:index, :show],
-                                         :scaptimony_scap_contents => [:index, :show],
-                                        }
-          permission :edit_compliance, {:scaptimony_arf_reports => [:destroy],
-                                        :scaptimony_policies => [:new, :create, :edit, :update, :destroy,
-                                          :select_multiple_hosts, :upldate_multiple_hosts],
-                                        :scaptimony_scap_contents => [:new, :create, :edit, :update]
-                                       }
+          permission :view_arf_reports, {:scaptimony_arf_reports         => [:index, :show, :parse, :auto_complete_search],
+                                         'api/v2/compliance/arf_reports' => [:index, :show],
+                                         :scaptimony_hosts               => [:show]
+                                      }
+          permission :edit_arf_reports, {:scaptimony_arf_reports         => [:destroy],
+                                         'api/v2/compliance/arf_reports' => [:destroy]}
+          permission :create_arf_reports, {'api/v2/compliance/arf_reports' => [:create]}
+
+          permission :view_policies, {:scaptimony_policies  => [:index, :show, :parse, :auto_complete_search],
+                                      :scaptimony_dashboard => [:index],
+                                      'api/v2/compliance/policies' => [:index, :show, :content]}
+          permission :edit_policies, {:scaptimony_policies          => [:new, :create, :edit, :update, :destroy, :scap_content_selected],
```

Comment:
Also we usually distinguish between edit and create permissions. In this case it's not that important as we don't support granular filters (that can be applied only for edit, not for create permission) but at least because of consistency. Same applies to edit permissions below.